components/CommunitySpotlight/CommunitySpotlightListings.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div class="subpage">
+    <div v-for="(item, index) in stories" :key="index">
+      <community-spotlight-item :story="item" />
+      <div v-if="index !== stories.length - 1 || inNews" class="seperator-path" />
+    </div>
+    <template v-if="inNews">
+      <nuxt-link
+        class="community-link mt-16"
+        :to="{
+          name: 'news-and-events-community-spotlight',
+        }"
+      >
+        View All Community Spotlights &gt;
+      </nuxt-link>
+    </template>
+  </div>
+</template>
+
+<script>
+import CommunitySpotlightItem from './CommunitySpotlightItem.vue'
+
+export default {
+  name: 'CommunitySpotlightListings',
+  components: {
+    CommunitySpotlightItem
+  },
+  props: {
+    stories: {
+      type: Array,
+      default: () => []
+    },
+    inNews: {
+      type: Boolean,
+      default: false
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import "@/assets/_variables.scss";
+
+.seperator-path {
```

Comment:
Not sure if this is desired or not. Looks a little weird in a full screen browser that the separator line doesn't match the width of the text. 
![image](https://user-images.githubusercontent.com/5529126/124756148-25a9b000-defa-11eb-99ac-4c886ca572f8.png)

Reply:
Yeah the separator is fixed width will fix this